### ISSUE TITLE
Write detailed documentation of adding new layer .

### ISSUE THREAD

**Issue by sagarpreet-chadha** [publiclab/leaflet-environmental-layers#243]
Quoting myself: 

> Write a detailed documentation for first timers to add new layers easily . Then let's create first timer issues of currently opened issues .

**Comment by crisner** [comment]
@sagarpreet-chadha Would this be like including an example on getting started and where to include them in the project? I haven't had much experience in writing documentation however I would like to give it a try if it is alright.

**Comment by sagarpreet-chadha** [comment]
Yes including examples and explaining library folder structure and use of each function in layer class. You can also add the challenges faced when you made the fire layer. Thank you!
